src/jobs/insert-changelog-latest-in-main-if-needed.yml

```diff
@@ -0,0 +1,27 @@
+description: >
+  If the release corresponds to a previous version, creates a PR to insert the release changelog into main's CHANGELOG.md.
+  Requires the fastlane lane `insert_changelog_latest_in_main_if_needed` to be defined in the Fastfile of the repository
+parameters:
+  ruby_version:
+    type: string
+    default: "3.2.0"
+    description: "Ruby version to use for the Docker image"
+  cache-version:
+    type: string
+    description: "String used to version the cache key for dependencies"
+  working_directory:
+    type: string
+    default: .
+docker:
+  - image: cimg/ruby:<< parameters.ruby_version >>
+shell: /bin/bash --login -o pipefail
+steps:
+  - checkout
+  - install-gem-mac-dependencies:
+      cache-version: <<parameters.cache-version>>
+      working_directory: <<parameters.working_directory>>
+  - setup-git-credentials
+  - trust-github-key
+  - run:
+      name: Insert changelog latest in main if needed
+      command: bundle exec fastlane insert_changelog_latest_in_main_if_needed
```

Comment:
this can be in the fastlane plugin right? It doesn't have to be defined in each SDK

Reply:
Oh right! Yeah, it requires each SDK to have the `insert_changelog_latest_in_main_if_needed` defined. The reason is that each SDK needs to pass certain parameters to the method defined in the plugin: `insert_changelog_of_older_version`. For example, for Capacitor, this would be the call:
```
insert_changelog_of_older_version(
  sdk_version: current_version_number,
  changelog_path: changelog_path,
  changelog_latest_path: changelog_latest_path,
  repo_name: repo_name,
  dry_run: dry_run
)
```
So I don't think this orb could just call the plugin's `insert_changelog_of_older_version` as the parameters depend on each SDK